book/src/remapping.md

```diff
@@ -83,7 +83,8 @@ There are three kinds of commands that can be used in keymaps:
   keys and used for movement and editing. A list of static commands is
   available in the [Keymap](./keymap.html) documentation and in the source code
   in [`helix-term/src/commands.rs`](https://github.com/helix-editor/helix/blob/master/helix-term/src/commands.rs)
-  at the invocation of `static_commands!` macro and the `TypableCommandList`.
+  at the invocation of `static_commands!` macro and [`helix-term/src/commands/typed.rs`](https://github.com/helix-editor/helix/blob/master/helix-term/src/commands/typed.rs)
```

Comment:
Actually let's move this to the bullet point below which talks about typable commands. I don't think we should've been mentioning the `TypableCommandList` here anyways since this bullet is about static commands that can't be invoked in command mode (`:`)